data/coalition/lunarium outfits.txt

```diff
@@ -6,7 +6,7 @@ outfit "Socket Reactor Core"
 	"outfit space" -53
 	"tower slots" 6
 	description "Mixing borrowed human designs with Coalition design philosophies, the Lunarium has started producing these modular reactor setups to give their ships the necessary energy to run all of their systems. The reactor core by itself provides little practical benefit, instead serving as the nerve center for the safe operation of the attached towers."
-	description
+#	description
```

Comment:
To be honest the empty description line can just be removed, doesn't have to be commented out.